lib/SqlConnection.py

```diff
@@ -0,0 +1,149 @@
+import os,sys
+import logging
+import logging.handlers
+from cloghandler import ConcurrentRotatingFileHandler
+from collections import deque
+from sqlalchemy import *
+from sqlalchemy.dialects.postgresql import ARRAY
+
+class PipelineSqlConnection:
+
+    def __init__(self,**kwargs):
+        self.logger = kwargs.get('logger',None)
+        if not self.logger:
+            self.initializeLogging(**kwargs)
+
+        self.host = kwargs.get('HOST','localhost')
```

Comment:
I modified initialization of the engine to match the uri in the config file but still rely on create_engine which accepts a string parameter.  create_engine does not require a url object.